WG/src/main/java/net/goldtreeservers/worldguardextraflags/wg/wrappers/WorldGuardCommunicator.java

```diff
@@ -1,5 +1,6 @@
 package net.goldtreeservers.worldguardextraflags.wg.wrappers;
 
+import net.goldtreeservers.worldguardextraflags.wg.handlers.*;
```

Comment:
I don't really like to use the wildcard

Reply:
My bad. My IDE automatically collapses multiple imports into wildcards, and I didn't have it turned off when opening the WGEF project. I'll fix that in the next commit.